docs/apache-airflow-providers-vertica/index.rst

```diff
@@ -73,7 +73,7 @@ Requirements
 PIP package                              Version required
 =======================================  ==================
 ``apache-airflow``                       ``>=2.2.0``
-``apache-airflow-providers-common-sql``
+``apache-airflow-providers-common-sql``  ``>=1.1.0``
```

Comment:
Wait, these say >= 1.1.0, but provider_dependencies says >= 1.2.0?

Reply:
Yeah. I have to re-run doc generation after some manual chnages 🤦

Reply:
Regenerated.